core/src/main/java/com/arcrobotics/ftclib/hardware/SimpleServo.java

```diff
@@ -115,5 +115,19 @@ public String getDeviceType() {
         String controller = servo.getController().toString();
         return "SimpleServo: " + port + "; " + controller;
     }
+    
+    private double toRadians(double angle, AngleUnit angleUnit) {
+        if (angleUnit == AngleUnit.DEGREES) {
+            return Math.toRadians(angle);
+        }
+        return angle;
+    }
+
+    private double fromRadians(double angle, AngleUnit angleUnit) {
+        if (angleUnit == AngleUnit.DEGREES) {
+            return Math.toDegrees(angle);
+        }
+        return angle;
```

Comment:
This is conditioned with DEGREES because the internal angles are all radians, right? So the `angleUnit` here is the desired output angle unit?
```suggestion
        return angleUnit == AngleUnit.DEGREES ? Math.toDegrees(angle) : angle;
```

Reply:
Kind of, the “fromRadians” method (in both the AngleUnit one and ours here) ALWAYS takes the angle parameter in radians, it performs a conversion from radians to the given AngleUnit, so if the AngleUnit parameter is “RADIANS” there’s no need for conversion since the angle is already in that unit, and if it’s “DEGREES”,  we perform the conversion from radians to degrees.

..or at least that was my reasoning lol, this is kinda confusing ngl